provider/tests/test_mgmt_cmd.py

```diff
@@ -0,0 +1,58 @@
+"""
+Unittests for testing management commands.
+"""
+import unittest
+import mock
+from django.core.management import CommandError, call_command
+from provider.management.commands.delete_expired_grant_tokens import Command
+
+
+class TestDeleteExpiredGrantTokens(unittest.TestCase):
+    """
+    Tests for parsing arguments for the `delete_expired_grant_tokens` management command.
+    """
+    def setUp(self):
+        super(TestDeleteExpiredGrantTokens, self).setUp()
+        self.command = Command()
+        self.command_name = 'delete_expired_grant_tokens'
+
+    @mock.patch('provider.oauth2.models.Grant.objects')
+    def test_no_args(self, objects):
+        objects.raw = mock.Mock(return_value=10)
+        self.command.handle(self.command_name)
+        objects.raw.assert_has_calls(
```

Comment:
So I tried out the SQL I wrote on a sqlite DB and confirmed that it's not compatible with the SQL supported by sqlite.

Then I tried out changing the mgmt command to use the Django ORM:
```
    def handle(self, *args, **options):
        """
        Deletes oauth2_grant tokens expired for days_back days, chunking the deletes to avoid long table/row locks.
        """
        days_back = options.get('days_back', self.DEFAULT_DAYS_BACK)
        if days_back < 0:
            raise CommandError('Only non-negative days back are allowed ({}).'.format(days_back))
        delete_date = datetime.now() - timedelta(days_back)
        log.info("STARTED: Deleting expired grant tokens older than %s.", delete_date)

        total_deletions = 0
        while True:
            with transaction.atomic():
                expired_grants = Grant.objects.filter(expires__lte=delete_date)[0:self.CHUNK_SIZE]
                delete_result = expired_grants.delete()
            deleted = delete_result[0]
            total_deletions += deleted
            if deleted == 0:
                # No more rows to delete.
                break
            else:
                log.info("Deleted %s expired tokens...", deleted)
        log.info("FINISHED: deleted %s expired grant tokens total.", total_deletions)
```

But as I suspected, Django (1.11) ORM refuses to DELETE with a specified LIMIT:
```
  File "/Users/jeskew/edx/src/django-oauth2-provider/provider/management/commands/delete_expired_grant_tokens.py", line 52, in handle
    delete_result = expired_grants.delete()
  File "/Users/jeskew/Envs/django-oauth2-provider/lib/python2.7/site-packages/django/db/models/query.py", line 598, in delete
    "Cannot use 'limit' or 'offset' with delete."
AssertionError: Cannot use 'limit' or 'offset' with delete.
```
So I can't actually run the raw SQL during the testing since tests use sqlite.

I propose that I leave this testing as is - and trust that SQL and MySQL are well-tested.